src/api/entities/User.ts

```diff
@@ -46,7 +50,6 @@ export class User extends BaseModel {
   declare lastName: string;
   declare phone?: string;
   declare role: UserRole;
-  declare participantId?: number | null;
```

Comment:
Optional: You could add a Participant[] here, similar to how Participant has a User[]. I'm still not 100% confident with Objection though so not sure if it will work without other changes though.

Reply:
I ended up adding in `Participant[]`!